packages/flutter_tools/lib/src/base/clock.dart

```diff
@@ -0,0 +1,16 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:quiver/time.dart';
+
+import 'context.dart';
```

Comment:
this is out of scope of this PR, and I defer to @johnmccutchan in general now about this since he owns the tool, but, I gotta say, I hope we can move away from the "context" approach. It's just fancy global variables. I'd much rather see explicit injection of dependencies, e.g. if FlutterVersion took a `clock` argument and defaulted to `const Clock()`.